### PR TITLE
Correct the keys for HTTP Metrics

### LINKED ISSUE #29189: A bug in our HTTP metrics for open telemetry
**Describe the bug**  
When registering HTTP metric attributes with open telemetry we use strings as keys, however this causes a mismatch with some AttributeKey<Long> in the TCK causing a test to fail. 

**Steps to Reproduce**  
Run the latest Open Telemetry 2.0 TCK

**Expected behavior**  
The TCK to pass

**Diagnostic information:**  
 - OpenLiberty Version: Trunk
 - Affected feature(s) MP Telemetry 2.0
 - Java Version: All
 - server.xml Not relevant.

**Additional context**

### PR DESCRIPTION
**Pull request by benjamin-confino**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Resolves #29189

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Review by Channyboy — DISMISSED** [review]

**Comment by benjamin-confino** [comment]
This PR makes us compatible with the latest version of the TCK, there was one build failure because we're now incompatible with the old version of the TCK. As the new version is not available in maven yet I added a PR to disable that test for now.

File: dev/io.openliberty.microprofile.telemetry.2.0.internal/src/io/openliberty/microprofile/telemetry20/internal/http/MPTelemetryHTTPMetricsAdapterImpl.java [hunk 1]
```diff
@@ -79,22 +89,22 @@ public void updateHttpMetrics(HttpStatAttributes httpStatAttributes, Duration du
...
+        attributesBuilder.put(HTTP_REQUEST_METHOD, httpStatAttributes.getRequestMethod());
+        attributesBuilder.put(URL_SCHEME, httpStatAttributes.getScheme());
 
-        Integer status = httpStatAttributes.getResponseStatus().orElse(-1);
-        attributesBuilder.put(OpenTelemetryConstants.HTTP_RESPONSE_STATUS_CODE, status == -1 ? "" : status.toString().trim());
+        Long status = Long.valueOf(httpStatAttributes.getResponseStatus().orElse(-1));
+        attributesBuilder.put(HTTP_RESPONSE_STATUS_CODE, status);
 
-        attributesBuilder.put(OpenTelemetryConstants.HTTP_ROUTE, httpStatAttributes.getHttpRoute().orElse(""));
+        attributesBuilder.put(HTTP_ROUTE, httpStatAttributes.getHttpRoute().orElse(""));
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal/src/io/openliberty/microprofile/telemetry20/internal/http/MPTelemetryHTTPMetricsAdapterImpl.java:98]: Not part of the change here, but should we be setting the route attribute if we don't have the data for it?

**Review by Azquelt — DISMISSED** [review]

**Review by Yas-Franco — DISMISSED** [review]

File: dev/io.openliberty.microprofile.telemetry.2.0.internal/src/io/openliberty/microprofile/telemetry20/internal/http/MPTelemetryHTTPMetricsAdapterImpl.java (hunk 1 above)
Comment (benjamin-confino) [reply dev/io.openliberty.microprofile.telemetry.2.0.internal/src/io/openliberty/microprofile/telemetry20/internal/http/MPTelemetryHTTPMetricsAdapterImpl.java:98]: Good question, I'll chase it up.

**Review by Yas-Franco — APPROVED** [review]
